bots/pr/src/test/java/org/openjdk/skara/bots/pr/PullRequestBotFactoryTest.java

```diff
@@ -122,6 +159,18 @@ public void testCreate() {
             assertTrue(integrators.contains("integrator1"));
             assertTrue(integrators.contains("integrator2"));
             assertTrue(pullRequestBot1.reviewCleanBackport());
+
+            var csrIssueBot1 = (CSRIssueBot) bots.get(3);
+            assertEquals(2, csrIssueBot1.repositories().size());
+            assertNotNull(csrIssueBot1.getPRBot("repo2"));
+            assertNotNull(csrIssueBot1.getPRBot("repo5"));
+            assertNotNull(csrIssueBot1.getPRBot("repo6"));
+
+            var csrIssueBot2 = (CSRIssueBot) bots.get(4);
+            assertEquals(1, csrIssueBot2.repositories().size());
+            assertNotNull(csrIssueBot2.getPRBot("repo2"));
+            assertNotNull(csrIssueBot2.getPRBot("repo5"));
+            assertNotNull(csrIssueBot2.getPRBot("repo6"));
```

Comment:
This doesn't look right. An instance of `CSRIssueBot` should only have access to `PullRequestBot` instances for repositories that share the same IssueProject.

Reply:
At first, I had the same idea with you and later I found it a little complex to implement it. But if we let all CSRIssueBots share a map, it would be very easy. If you think it's necessary to change, I will fix it.

Reply:
Yes, at least I will change the test here.